tests/format/js/arrow-call/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -194,9 +193,8 @@ func(
 );
 
 promise.then((result) =>
-  result.veryLongVariable.veryLongPropertyName > someOtherVariable
-    ? "ok"
-    : "fail",
+  result.veryLongVariable.veryLongPropertyName > someOtherVariable ? "ok"
+  : "fail",
```

Comment:
And this.

Reply:
Agreed, the previous version felt better